src/map/clif.cpp

```diff
@@ -13840,6 +13845,31 @@ void clif_parse_GuildChangeEmblem(int fd,struct map_session_data *sd){
 	guild_change_emblem(sd, emblem_len, (const char*)emblem);
 }
 
+/// Request to update the guild emblem id (version, according to Gravity)
+/// 0b46 <guild id>.L <version>.L
+void clif_parse_GuildChangeEmblem2(int fd, struct map_session_data* sd) {
+	nullpo_retv(sd);
+
+#if PACKETVER >= 20190724
+	const PACKET_CZ_GUILD_EMBLEM_CHANGE2* p = (PACKET_CZ_GUILD_EMBLEM_CHANGE2*)RFIFOP(fd, 0);
+	guild* g = sd->guild;
+
+	if (g == nullptr || g->guild_id != p->guild_id)
+		return;
+
+	int version = p->version;
```

Comment:
Can just do away with the assignment and just use `p->version` below as the argument for `guild_change_emblem_version`.